constellation-app/constellation
# Delimited File Importer - Option to insert column headers

### Description of the Change
An option `Insert Column Headers` is added in the Delimited File Importer source pane. When importing from delimited files that don't contain column headings, the option can be enabled to insert column headers as `Col1`, `Col2`, `Col3` etc.




### Alternate Designs
n/a


### Why Should This Be In Core?
Better UX, as without this, the first row will be skipped as the column headers.


### Benefits
It can now import from files with no column headers.


### Possible Drawbacks
n/a


### Verification Process

1. Create a CSV file that doesn't contain headers, e.g.:
5,6,7,2009-12-05 23:59:59,2009-12-1
1,1,1,2009-12-15 23:59:59,2009-12-12
a,a,a,2009-12-25 23:59:59,2009-12-31

2. Enable the check box and import it.
3. Confirm the columns are inserted and the first row of real data is not lost.



### Applicable Issues
https://github.com/constellation-app/constellation/issues/573
